Lib/concurrent/futures/process.py

```diff
@@ -627,6 +627,18 @@ class BrokenProcessPool(_base.BrokenExecutor):
     while a future was in the running state.
     """
 
+class _TerminateOrKillOperation(enum.Enum):
```

Comment:
use of the enum module and an enum for this may be overkill for this internal API of the stdlib... but i'm not going to block the PR on that - if desired for import time performance reasons (enum is frowned upon due it's overblown code complexity) it can be refactored to just a couple of constant values later.

Reply:
Its sort of unfortunate to have to not use other parts of the stdlib because of import timings, but I understand. I'll leave this for now as is unless someone has a performance oriented reason to swap to constants/a dict.

Reply:
1. Maybe use an IntEnum instead of an enum.Enum?
2. I'm not very fond of the use of the enumeration here as well but I won block this one. Wuld it be possible to already know how long the import now takes with -X importtime?